be/src/column/fixed_length_column_base.cpp

```diff
@@ -49,6 +50,16 @@ void FixedLengthColumnBase<T>::append_value_multiple_times(const Column& src, ui
     }
 }
 
+template <typename T>
+void FixedLengthColumnBase<T>::fill_default(const Filter& filter) {
+    T val = DefaultValueGenerator<T>::next_value();
```

Comment:
default value , using T(0) is just OK.  next_value is weird, it is not a iterator or generator. it would be better if we use 
```
// common case
template <typename T, typename=guard::Guard> 
struct DefaultValue{
static constexpr T default_value = T(0);
}
// special case
template <typename T> 
struct DefaultValue<T, FloatTypeGuard> {
   static constexpr T default_value<T, FloatGuard> = T(0.0);
}
```

Reply:
What's exactly the difference? Or just a naming concern?